docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -29,6 +29,7 @@ The following table lists the available UI components:
 | [TextLabel](./textlabel.md)          | A text label that renders a short text string. | `TextLabel`                |
 | [Toast](./nui-components/Toast.md) | A toast is used to provide simple messages. | `Toast`                |
 | [VideoView](./videoview.md)          | A video view that controls and displays video playback. | `VideoView`                 |
+| [Widget](./widget.md)                | A widget is used to create light Tizen .NET applications on the home screen. | `Widget` |
```

Comment:
Question.

I can see /nui-components folder under /nui.

Some "nui-components" such as Button, Loading or etc are grouped under docs/application/dotnet/guides/nui/nui-components/

For widget, was there any reason that we don't put it under the nui-components/ folder?

Reply:
First i put it in the current folder because it was simiral to "View"
but i think this folder is more reasonable.
https://docs.tizen.org/application/dotnet/guides/applications/nui-widget-app

how about your think?

Reply:
With your opinion, I checked the pages, and got same conclusion with you.

Based on the API reference, the NUI Widget Application seems closer to be placed under guide/applications.

Maybe we need more modification to move this page below guide/applications,
but would you change the location of this page as that place seems more proper?